src/konfo_backend/external/schema/koulutus_metadata.clj

```diff
@@ -44,49 +44,73 @@
     |        opintojenLaajuusyksikko:
     |          type: object
     |          $ref: '#/components/schemas/OpintojenLaajuusyksikko'
+    |    KorkeakouluMetadata:
+    |      allOf:
+    |        - $ref: '#/components/schemas/KoulutusMetadata'
+    |      properties:
+    |        kuvauksenNimi:
+    |          type: object
+    |          description: Koulutuksen kuvaukseni nimi eri kielillä. Kielet on määritetty koulutuksen kielivalinnassa.
+    |          allOf:
+    |            - $ref: '#/components/schemas/Nimi'
+    |       tutkintonimike:
+    |         type: array
+    |         properties:
+    |           koodiUri:
+    |             type: string
+    |             example: tutkintonimikekk_110#2
+    |             description: Tutkintonimikkeen koodi URI. Viittaa [koodistoon](https://virkailija.testiopintopolku.fi/koodisto-ui/html/koodisto/tutkintonimikkeet/2)
+    |           nimi:
+    |             type: object
+    |             description: Tutkintonimikkeem nimi eri kielillä.
+    |             example: {\"fi\": \"Tutkintonimike suomeksi\"}
+    |             allOf:
+    |               - $ref: '#/components/schemas/Nimi'
+    |       opintojenLaajuus:
+    |         type: object
+    |         properties:
+    |           koodiUri:
+    |             type: string
+    |             example: opintojenlaajuus_40
+    |             description: Tutkinnon laajuus. Viittaa koodistoon [koodistoon](https://virkailija.testiopintopolku.fi/koodisto-ui/html/koodisto/opintojenlaajuus/1)
+    |           nimi:
+    |             type: object
+    |             description: Tutkinnon laajuuden eri kielillä.
+    |             example: {\"fi\": \"Tutkinnon laajuus suomeksi\"}
+    |             allOf:
+    |               - $ref: '#/components/schemas/Teksti'
```

Comment:
Oletko kokeillut, että swagger on ehjä? Ajamalla ./lein run ja avaamalla swaggerin selaimella... nimittäin tämä ei mun nähdäkseni ole nyt eheän näköinen, kun arrayssa pitäisi olla items ja se alla type: object. Projektista löytyy kyllä esimerkkejä.